pkg/cli/azure/provider.go

```diff
@@ -16,20 +16,33 @@ limitations under the License.
 
 package azure
 
+// AzureCredentialKind - Azure credential kinds supported.
+type AzureCredentialKind string
+
 const (
 	// ProviderDisplayName is the text used in display for Azure.
-	ProviderDisplayName = "Azure"
+	ProviderDisplayName                                     = "Azure"
+	AzureCredentialKindWorkloadIdentity AzureCredentialKind = "WorkloadIdentity"
+	AzureCredentialKindServicePrincipal AzureCredentialKind = "ServicePrincipal"
 )
```

Comment:
Where is managed identity auth used https://github.com/radius-project/radius/blob/a6e749d185db8785e8a0710d8ba155da1d72ebed/pkg/azure/armauth/auth.go#L34?

Reply:
I don't think we use this anymore, looks like it was added 3 years ago.